server/internal/notifications/index.ts

```diff
@@ -102,16 +123,27 @@ class NotificationSystem {
       },
     });
 
-    const res: Promise<void>[] = [];
-    for (const user of users) {
-      res.push(this.push(user.id, notificationCreateArgs));
-    }
-    // wait for all notifications to pass
-    await Promise.all(res);
+    await this._pushMany(notificationCreateArgs, users);
   }
 
+  /**
+   * Send a notification to all system level users
+   * @param notificationCreateArgs
+   * @returns
+   */
   async systemPush(notificationCreateArgs: NotificationCreateArgs) {
-    return await this.pushAll(notificationCreateArgs);
+    const users = await prisma.user.findMany({
+      where: {
+        id: { not: "system" },
+        // no reason to send to any users other then admins rn
```

Comment:
Not if users get promoted to admin. I think it's okay either way though

Reply:
That's actually exactly why i did it, didn't make sense that new admins would be able to the entire notification history